### PR TITLE
Feature: Add modal and sign in mobile screen

### PR DESCRIPTION
**Pull request by VladyslavP**

### CODE REVIEW

File: src/mobile/styles/components/Modal.pcss [hunk 1]
```diff
@@ -1,29 +1,65 @@
...
+
   &--open {
     height: 100vh;
   }
 
   &__block {
-    background: white;
+    position: absolute;
+    margin-top: 20px;
+    background-color: #1E2841;
```
Comment (oyershov) [inline src/mobile/styles/components/Modal.pcss:52]: use color variable here

File: src/mobile/styles/components/Modal.pcss [hunk 2]
```diff
@@ -1,29 +1,65 @@
...
-  height: 0;
   left: 0;
-  position: fixed;
   right: 0;
+  height: 0;
   z-index: 999;
 
+  &__header {
+    height: 36px;
+    background: #28334E;
```
Comment (oyershov) [inline src/mobile/styles/components/Modal.pcss:13]: use color variable here

File: src/mobile/components/Modal/Modal.tsx [hunk 3]
```diff
@@ -1,21 +1,39 @@
 import classnames from 'classnames';
 import * as React from 'react';
-import { useIntl } from 'react-intl';
+import { CloseIcon } from '../../../assets/images/CloseIcon';
+import { ArrowIcon } from '../../../containers/ToolBar/icons/ArrowIcon';
 
 const Modal = props => {
+    const [shouldAnimate, setShouldAnimate] = React.useState(false);
```
Comment (oyershov) [inline src/mobile/components/Modal/Modal.tsx:7]: use styles for animation
```transition``` and ```position```

File: src/components/SignIn/index.tsx [hunk 4]
```diff
@@ -34,154 +38,177 @@ export interface SignInProps {
...
+                    </div>
+                    {!isMobileDevice && renderForgotButton()}
+                    {isMobileDevice && renderRegister()}
+                </div>
+            </div>
+        </form>
+    );
+};
+
+export {
```
Comment (oyershov) [inline src/components/SignIn/index.tsx:212]: Use React.memo()

File: src/mobile/styles/components/Modal.pcss (hunk 1 above)
Comment (VladyslavP) [reply src/mobile/styles/components/Modal.pcss:52]: lol, didn't notice, thanks!

File: src/mobile/components/Modal/Modal.tsx (hunk 3 above)
Comment (VladyslavP) [reply src/mobile/components/Modal/Modal.tsx:7]: it is in order to apply animation

**Pull request merged**